can_read_write.py

```diff
@@ -0,0 +1,94 @@
+# This scipt must run on Pi startup
+
+import can
+import time
+import os
+import sys
+import json
+from threading import Thread
+
+# info handled; specified by https://docs.google.com/document/d/1kUU54jQZAB9nwCM-iA96Kj0fXJ6RNw9nAcBffzji5cU/edit
+# full list of PIDs at https://en.wikipedia.org/wiki/OBD-II_PIDs
+PIDS = {'VEHICLE_CHARGE': 0x2F,
+        'VEHICLE_SPEED': 0x0D,
+}
+
+
+PID_REQUEST = 0x7DF
```

Comment:
Assuming I'm understanding this naming correctly, I think `REQUEST_PARAM` would be a better name. Calling it `PID_REQUEST` makes me think it's sort of analogous to the things in `PIDS`, which seems to not be the case.